pkg/apiserver/experiment/experiment.go

```diff
@@ -159,7 +159,7 @@ func (s *ScopeInfo) parseSelector() v1alpha1.SelectorSpec {
 
 // TargetInfo defines the information of target objects.
 type TargetInfo struct {
-	Kind         string           `json:"kind" binding:"required,oneof=PodChaos NetworkChaos IOChaos KernelChaos TimeChaos StressChaos"`
+	Kind         string           `json:"kind" binding:"required,oneof=PodChaos NetworkChaos IoChaos KernelChaos TimeChaos StressChaos"`
```

Comment:
I rename `IOChaos` to `IoChaos` here since we are using `IoChaos` in io chaos kind. However I think maybe we should rename it to `IOChaos` later.

```
apiVersion: pingcap.com/v1alpha1
kind: IoChaos
metadata:
  creationTimestamp: "2020-05-21T19:50:03Z"
  finalizers:
  - chaos-testing/etcd-577c6ffdd4-wcqm6
  generation: 32
  name: iochaos-1
  namespace: chaos-testing
  resourceVersion: "81894893"
  selfLink: /apis/pingcap.com/v1alpha1/namespaces/chaos-testing/iochaos/iochaos-1
  uid: a8a8ea5a-02dc-4dae-957d-fb0ff8714aca
```